polar-core/src/terms.rs

```diff
@@ -451,6 +451,16 @@ impl Term {
             None
         }
     }
+
+    // TODO(gj): what happens if the user registers a class as `Actor` or `Resource`?
```

Comment:
Agree that it's probably an uncommon case; I think the three options are:

1. Leave them as `Actor` / `Resource` and throw an error if folks try to register constants with those names.
2. Namespace as `Oso::Actor` / `Oso::Resource` or w/e to reduce the likelihood of collisions but otherwise same situation as (1).
3. Wait to name the actor/resource "unions" until after all application classes have been registered, then decide on a name for each that won't conflict with any application class names.

Reply:
I think 1 is the most straight-forward.